src/search/mod.rs

```diff
@@ -135,7 +135,8 @@ impl Document for ResultSearchMod {
 
 pub async fn search_for_mod(info: &SearchRequest) -> Result<SearchResults, SearchError> {
     let address = &*dotenv::var("MEILISEARCH_ADDR")?;
-    let client = Client::new(address, "");
+    let key = &*dotenv::var("MEILISEARCH_KEY")?;
+    let client = Client::new(address, key);
```

Comment:
This may be a good time to move the address and key into a separate structure so that they don't have to be looked up on every request.  We should be able to put it in a actix_web `Data` structure that's cached for each thread.  We could also just cache the meilisearch `Client`, but that may be a bit more complicated.